src/components/results/securityanalysis/security-analysis-result-utils.ts

```diff
@@ -131,36 +124,130 @@ export const flattenNmKResultsConstraints = (
+        field: 'limitName',
+        isSortable: true,
+        isFilterable: true,
+        filterParams: {
+            filterUIType: FILTER_UI_TYPES.TEXT,
+            filterComparators: [
+                FILTER_TEXT_COMPARATORS.STARTS_WITH,
+                FILTER_TEXT_COMPARATORS.CONTAINS,
+            ],
+        },
+    }),
+
+    makeColumn({
         headerName: intl.formatMessage({ id: 'Limit' }),
         field: 'limit',
+        numeric: true,
+        isSortable: true,
+        isFilterable: true,
         valueFormatter: (params: ValueFormatterParams) =>
             params.data?.limit?.toFixed(1),
-    },
-    {
-        headerName: intl.formatMessage({ id: 'Value' }),
+        filterParams: {
+            filterUIType: FILTER_UI_TYPES.NUMBER,
+            filterComparators: [
+                FILTER_NUMBER_COMPARATORS.NOT_EQUAL,
+                FILTER_NUMBER_COMPARATORS.GREATER_THAN_OR_EQUAL,
+                FILTER_NUMBER_COMPARATORS.LESS_THAN_OR_EQUAL,
+            ],
+        },
+    }),
+    makeColumn({
+        headerName: intl.formatMessage({ id: 'CalculatedValue' }),
         field: 'value',
+        isSortable: true,
+        isFilterable: true,
         valueFormatter: (params: ValueFormatterParams) =>
             params.data?.value?.toFixed(1),
-    },
-    {
+        filterParams: {
+            filterUIType: FILTER_UI_TYPES.NUMBER,
+            filterComparators: [
+                FILTER_NUMBER_COMPARATORS.NOT_EQUAL,
+                FILTER_NUMBER_COMPARATORS.GREATER_THAN_OR_EQUAL,
+                FILTER_NUMBER_COMPARATORS.LESS_THAN_OR_EQUAL,
+            ],
+        },
+    }),
+
+    makeColumn({
         headerName: intl.formatMessage({ id: 'Loading' }),
         field: 'loading',
+        isSortable: true,
+        isFilterable: true,
         valueFormatter: (params: ValueFormatterParams) =>
-            params.data.loading?.toFixed(1),
-    },
+            params.data?.loading?.toFixed(1),
+        filterParams: {
+            filterUIType: FILTER_UI_TYPES.NUMBER,
+            filterComparators: [
+                FILTER_NUMBER_COMPARATORS.NOT_EQUAL,
+                FILTER_NUMBER_COMPARATORS.GREATER_THAN_OR_EQUAL,
+                FILTER_NUMBER_COMPARATORS.LESS_THAN_OR_EQUAL,
+            ],
+        },
+    }),
+
+    makeColumn({
+        headerName: intl.formatMessage({
+            id: 'Overload',
+        }),
+        field: 'acceptableDuration',
+        cellRenderer: (value: ValueFormatterParams) =>
+            convertMillisecondsToMinutesSeconds(value.data.acceptableDuration),
+        numeric: true,
+        isSortable: true,
+        isFilterable: true,
+        filterParams: {
+            filterUIType: FILTER_UI_TYPES.NUMBER,
+            filterComparators: [
+                FILTER_NUMBER_COMPARATORS.NOT_EQUAL,
+                FILTER_NUMBER_COMPARATORS.GREATER_THAN_OR_EQUAL,
+                FILTER_NUMBER_COMPARATORS.LESS_THAN_OR_EQUAL,
```

Comment:
```suggestion
                Object.values(FILTER_NUMBER_COMPARATORS);
```

to do for each number columns

Reply:
done